ui/components/app/modals/new-account-modal/new-account-modal.component.js

```diff
@@ -43,9 +44,9 @@ export default class NewAccountModal extends Component {
         <div className="new-account-modal__content">
           <div className="new-account-modal__content__header">
             {t('newAccount')}
-            <button
-              className="fas fa-times new-account-modal__content__header-close"
-              title={t('close')}
+            <ButtonIcon
+              className="new-account-modal__content__header-close"
```

Comment:
Could you remove remove CSS in favour of Box props here? I don't think we need to include any of these styles. Maybe `color`?

![Screenshot 2023-02-06 at 3 21 20 PM](https://user-images.githubusercontent.com/8112138/217109477-40d758bf-f42d-4b17-8879-ed19949e0788.png)